internal/store/pod.go

```diff
@@ -1362,6 +1364,32 @@ func createPodStartTimeFamilyGenerator() generator.FamilyGenerator {
 	)
 }
 
+func createPodStatusContainersReadyTimeFamilyGenerator() generator.FamilyGenerator {
```

Comment:
Could you add a test for both of these, thank you!

Reply:
Added, thank you!